### ISSUE TITLE
Is there a support for CenterCrop like in PyTorch transforms

### ISSUE THREAD

**Issue by bkanaki** [aleju/imgaug#193]
Hi @aleju , thank you for this great contribution. I really like using this library when I train my deep learning models in Keras.

Recently, I was wondering if this library supports the [CenterCrop](https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.CenterCrop) like in TorchVision transforms? 

In the docs, I only see CropAndPad which needs either pixel values or percentage to crop. But, a feature like center crop is really helpful when using many deep learning pipelines. 

Can this be added to the existing set if transformations here?

Please let us know! Thanks.

**Comment by aleju** [comment]
CenterCrop itself is not yet available - still have to add an augmenter for that.
But you should be able to emulate its behaviour with the existing ones.
Here is an example:
```python
import imgaug as ia
from imgaug import augmenters as iaa
from imgaug import parameters as iap

def CenterCrop(height, width):
    crop = iaa.CropToFixedSize(height=height, width=width)
    crop.position = (iap.Deterministic(0.5), iap.Deterministic(0.5))
    pad = iaa.PadToFixedSize(height=height, width=width, pad_mode=ia.ALL, pad_cval=(0, 255))
    pad.position = (iap.Deterministic(0.5), iap.Deterministic(0.5))
    return iaa.Sequential([crop, pad])

seq = iaa.Sequential([CenterCrop(height=224, width=224), ...])
```
Note that this also pads to the expected size (which I think pytorch's CenterCrop does) and uses several different padding methods (while I think that pytorch's CenterCrop only uses zero-padding).
